ocdata/utils/vasp.py

```diff
@@ -212,19 +224,19 @@ def xml_to_tuples(xml='vasprun.xml'):
     return images
 
 
-def xml_to_traj(xml='vasprun.xml'):
-    '''
+def xml_to_traj(xml="vasprun.xml"):
+    """
```

Comment:
not used - remove

Reply:
Updated in https://github.com/Open-Catalyst-Project/Open-Catalyst-Dataset/commit/ac78be51e6108b9d69c071386844617d715ff4ab